### PR TITLE
Pilot PR for MTU migration (OVN)

### PR DESCRIPTION
**Pull request by anuragthehatter**
The flow looks like ```Install cluster -> decrease MTU migration by 400 -> increase MTU migration by 200``` and we can run e2e tests post that. The final MTU on cluster will not be same as we see.
 
As per dev, upgrade prepare and upgrade checks are not needed but running some few e2e tests post final migration should be enough. So we can introduce e2e test post increase step to make sure nothing breaks. If CI budget permits, we can just run ovn e2e suite post that.

Consider as a Pilot PR and we can do same for SDN backports.

### CODE REVIEW

File: ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse-4.12.yaml [hunk 1]
```diff
@@ -66,11 +66,11 @@ tests:
   steps:
     cluster_profile: alibabacloud-cn-qe
     workflow: cucushift-installer-rehearse-alibabacloud-ipi
-- as: installer-rehearse-gcp
+- as: installer-rehearse-gcp-mtu-migrate
```
[the remark this answers is not in the document]
Comment (anuragthehatter) [reply ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse-4.12.yaml]: Thanks. Agree. taken care of

**Comment by jianlinliu** [comment]
As usual, pls squash multiple commits into one.

**Comment by anuragthehatter** [comment]
> As usual, pls squash multiple commits into one.

This one seems hard to squash. My git log shows only commits pre master rebase. Might mess thisgs up. I can create a new PR in liu of this one. Thanks

**Comment by anuragthehatter** [comment]
Opened https://github.com/openshift/release/pull/32298

**Pull request closed without merging**